### ISSUE TITLE
9.2 is micro-infra incompatible, downgrade to 9.1 shows error

### ISSUE THREAD

**Issue by johnykov** [Codearte/accurest#148]
Error message is:

```
Execution failed for task ':generateAccurest'.
> Cannot get property 'parameters' on null object
```

and the reason is incompatible `urlPath` in 9.2 and `url` in 9.1 parameter in contract

**Comment by mariuszs** [comment]
+1

**Comment by jkubrynski** [comment]
@hanskoff can you describe what you mean by incompatible? The only change in 9.2 was NPE fix

**Comment by szpak** [comment]
Not only. Migration to the new Wiremock is also a significant change :)
https://github.com/Codearte/accurest/commit/b4689efd363a531635afd1b12254f8671ba622c8

**Comment by johnykov** [comment]
accurest 9.2  is incompatible with micro-infra because of wiremock 2.0.2 :-1:

**Comment by mariuszs** [comment]
this was because of micro-infra bug, already fixed in latest version micro-infra-spring 0.9.6
